src/filters/type.js

```diff
@@ -12,12 +12,14 @@ const markdownItResponsive = require('@gerhobbelt/markdown-it-responsive');
 
 const { taxonomy } = require('#/taxonomy');
 
-const responsiveConfig = taxonomy.srcset.map((size) => ({
-  width: Number(size),
-  rename: {
-    suffix: `-${size}`,
-  },
-}));
+const responsiveConfig = taxonomy.srcset
+  .map((size) => ({
+    width: size,
+    rename: {
+      suffix: `-${size}`,
+    },
+  }))
+  .concat([{}]);
```

Comment:
@mirisuzanne @duggiemitchell This empty object means that the fallback original image is also processed and passed through along with the new resized ones.

Reply:
That's great, but without supplying the width, it results in[ invalid `srcset`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-srcset):

- "If no descriptor is specified, the source is assigned the default descriptor of 1x."
- "It is incorrect to mix width descriptors and pixel density descriptors in the same srcset attribute."

I'd love to talk through some of the isseues here, around supplying a width to markdown. Or we can try to move away from md images entirely to using `embed.img()` macro.

Reply:
I don't follow... All this does is process the default images through the optimizer as well -- this change shouldn't impact the `srcset` attr at all.

Reply:
well maybe it's a problem with the plugin then? The result of a md image is not currently valid:

```html
<img src="/assets/images/blog/2016/cssconf/sd.jpg" alt="Sarah Drasner speaking" srcset="/assets/images/blog/2016/cssconf/sd-480.jpg 480w, /assets/images/blog/2016/cssconf/sd-960.jpg 960w, /assets/images/blog/2016/cssconf/sd-1280.jpg 1280w, /assets/images/blog/2016/cssconf/sd-2240.jpg 2240w, /assets/images/blog/2016/cssconf/sd.jpg ">
```